web_src/less/_admin.less

```diff
@@ -1,4 +1,14 @@
 .admin {
+  &.hooks .list {
+    > .item {
+      &:not(:first-child) {
```

Comment:
Could also `.item + .item` for a more robust selector but I think there are more cases like this in the codebase.